### PR TITLE
Fix overlapping group symbols

### LINKED ISSUE #1320: staffGrp symbol overlaps
In the following example, the brace (highlighted in red) for the first and second violins are overlapping with the system bracket:

Screen Shot 2020-02-22 at 15 01 10

Here is an example where a brace is used at the same time as a bracket:
http://imslp.simssa.ca/files/imglnks/usimg/0/0c/IMSLP52624-PMLP01586-Beethoven_Werke_Breitkopf_Serie_1_No_5_Op_67.pdf

Screen Shot 2020-02-22 at 15 02 07

Notice that the brace does not touch the bracket (in a manner similar to when the brace is attached directly to the system without the brace).

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt />
  </fileDesc>
  <encodingDesc>
   <appInfo>
    <application isodate="2020-02-22T15:03:00" version="2.6.0-dev-fd1d45b">
     <name>Verovio</name>
     <p>Transcoded from Humdrum</p>
    </application>
   </appInfo>
  </encodingDesc>
  <workList>
   <work>
    <title />
   </work>
  </workList>
  <extMeta>
   <frames xmlns="http://www.humdrum.org/ns/humxml">
    <metaFrame n="10" token="!!!system-decoration: [{s1,s2},s3,s4]" xml:id="L11">
     <frameInfo>
      <startTime float="4" />
      <frameType>reference</frameType>
      <referenceKey>system-decoration</referenceKey>
      <referenceValue>[{s1,s2},s3,s4]</referenceValue>
     </frameInfo>
    </metaFrame>
   </frames>
  </extMeta>
 </meiHead>
 <music>
  <body>
   <mdiv xml:id="mdiv-0000001487912434">
    <score xml:id="score-0000002044692570">
     <scoreDef xml:id="scoredef-0000001233355154" midi.bpm="400">
      <staffGrp xml:id="staffgrp-0000001875594997" symbol="bracket">
       <staffGrp xml:id="staffgrp-0000000212660266" symbol="brace" bar.thru="false">
        <staffDef xml:id="staffdef-0000000194489244" n="1" lines="5">
         <label xml:id="label-L3F4">Violin I</label>
         <clef xml:id="clef-L4F4" shape="G" line="2" />
         <keySig xml:id="keysig-L5F4" sig="0" />
         <meterSig xml:id="metersig-L6F4" count="4" unit="4" />
        </staffDef>
        <staffDef xml:id="staffdef-0000001511189038" n="2" lines="5">
         <label xml:id="label-L3F3">Violin II</label>
         <clef xml:id="clef-L4F3" shape="C" line="3" />
         <keySig xml:id="keysig-L5F3" sig="0" />
         <meterSig xml:id="metersig-L6F3" count="4" unit="4" />
        </staffDef>
       </staffGrp>
       <staffGrp xml:id="staffgrp-0000000768302054" bar.thru="false">
        <staffDef xml:id="staffdef-0000000520235216" n="3" lines="5">
         <label xml:id="label-L3F2">Viola</label>
         <clef xml:id="clef-L4F2" shape="F" line="4" />
         <keySig xml:id="keysig-L5F2" sig="0" />
         <meterSig xml:id="metersig-L6F2" count="4" unit="4" />
        </staffDef>
        <staffDef xml:id="staffdef-0000001317356624" n="4" lines="5">
         <label xml:id="label-L3F1">Violoncello</label>
         <clef xml:id="clef-L4F1" shape="F" line="4" />
         <keySig xml:id="keysig-L5F1" sig="0" />
         <meterSig xml:id="metersig-L6F1" count="4" unit="4" />
        </staffDef>
       </staffGrp>
      </staffGrp>
     </scoreDef>
     <section xml:id="section-L1F1">
      <measure xml:id="measure-L1" n="0">
       <staff xml:id="staff-0000001740634549" n="1">
        <layer xml:id="layer-L1F4N1" n="1">
         <mRest xml:id="mrest-L8F4" />
        </layer>
       </staff>
       <staff xml:id="staff-0000001822625609" n="2">
        <layer xml:id="layer-L1F3N1" n="1">
         <mRest xml:id="mrest-L8F3" />
        </layer>
       </staff>
       <staff xml:id="staff-0000001161869655" n="3">
        <layer xml:id="layer-L1F2N1" n="1">
         <mRest xml:id="mrest-L8F2" />
        </layer>
       </staff>
       <staff xml:id="staff-0000000474489414" n="4">
        <layer xml:id="layer-L1F1N1" n="1">
         <mRest xml:id="mrest-L8F1" />
        </layer>
       </staff>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

### PR DESCRIPTION
**Pull request by rettinghaus**
closes #1320

### CODE REVIEW

**Comment by rettinghaus** [comment]
example:
score

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [unreleased]
+* Fix overlapping group symbols (@rettinghaus)
```
Comment (lpugin) [inline CHANGELOG.md]: Keep small bug fixes below in the list

Comment (rettinghaus) [reply CHANGELOG.md]: Done

**Comment by rettinghaus** [comment]
Now also working correctly with scaled staffs.
old:
score2-old
new:
score2-new

**Pull request merged**